kvbc/test/categorization/kv_blockchain_test.cpp

```diff
@@ -478,6 +476,159 @@ TEST_F(categorized_kvbc, link_state_transfer_chain) {
   }
 }
 
+TEST_F(categorized_kvbc, creation_of_category_type_cf) {
+  KeyValueBlockchain block_chain{db, true};
+  ASSERT_TRUE(db->hasColumnFamily(detail::ST_CHAIN_CF));
```

Comment:
Should we check for the `CAT_ID_TYPE` instead?

Reply:
copy paste err probably :)